client/src/components/form/form.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext } from 'react';
+
+const FormHeader: React.FC<React.HTMLAttributes<HTMLHeadingElement>> = ({ children, ...props }) => {
```

Comment:
[nit] The name `FormHeader` is pretty redundant. We're already in the context of `Form`.

Reply:
I want to say yes and no. A part of reasoning for the form component is to control styling. This way, all headers will be like this unless it's a special case. The <Form/> itself I believe should just be as simple as possible.